velox/duckdb/conversion/DuckParser.cpp

```diff
@@ -463,7 +504,10 @@ std::shared_ptr<const core::IExpr> parseCaseExpr(
     inputs.emplace_back(elseExpr);
   }
 
-  return callExpr("switch", std::move(inputs), getAlias(expr));
+  return callExpr(
+      toFullFunctionName("switch", options.functionPrefix),
```

Comment:
Would it make sense to move toFullFunctionName inside callExpr?

Reply:
Yes, this makes sense.